sdk/storage/azure-storage-blob/src/main/java/com/azure/storage/blob/sas/package-info.java

```diff
@@ -0,0 +1,7 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+/**
+ * Package containing sas classes used by Azure Storage Blobs.
```

Comment:
upper-case the 'sas' and expand it in brackets to help users understand what this means.

Reply:
SAS [shared access signature] or SAS (shared access signature)?